install.sh

```diff
@@ -87,24 +87,24 @@ if [ ! -z "$TARBALL" ] ; then
     else
         # Calling script may have already set DDEV_INSTALL_DIR, otherwise we respect and use it.
         # Otherwise, try to use /usr/local/bin and then /usr/bin
-        if [ ! -z "${DDEV_INSTALL_DIR}" ]; then
+        if [ ! -z "${DDEV_INSTALL_DIR:-}" ]; then
```

Comment:
I ran into this on the build on master (ddev.exe exists in my path in Git-Bash) as I installed it in step 3 and this is step 5. 

```
\033[33mPLease use the ddev_windows_installer provided with this package to install ddev\033[0m
./install.sh: line 85: DDEV_INSTALL_DIR: unbound variable
```

Reply:
Ah, it's because TARBALL exists for the test bot, but we don't want it in the real case.

Reply:
Patched install.sh with the commit changes and it worked for me.